pom.xml

```diff
@@ -17,15 +14,19 @@
     <packaging>hpi</packaging>
     <name>Rebuilder</name>
 
+    <properties>
+        <!-- Since rebuild plugin is using some functions available in
+        Jenkins version 1.405 ,version is updating from 1.400 to 1.405-->
+        <jenkins.version>1.481</jenkins.version>
+        <jenkins-test-harness.version>1.481</jenkins-test-harness.version>
+        <java.level>5</java.level>
```

Comment:
Jenkins < 1.520 targets Java5.
And rebuild plugin doesn't use features of Java6, anyway.

Reply:
I would probably bump it to something reasonably new like 1.609.3 (Java 6). It pulls in more annotations in the core && seems to be reasonable for testing purposes. According to the usage stats, it's pretty fine: http://stats.jenkins.io/pluginversions/rebuild.html